osu.Game/Configuration/SettingSourceAttribute.cs

```diff
@@ -31,7 +31,7 @@ public class SettingSourceAttribute : Attribute, IComparable<SettingSourceAttrib
     {
         public LocalisableString Label { get; }
 
-        public string Description { get; }
+        public LocalisableString Description { get; }
```

Comment:
This one's a little indirect. Without it, the following crashes lower down:

https://github.com/ppy/osu/blob/ee6cf4396a0d6e57f2857967b15b6db64b7379b5/osu.Game/Configuration/SettingSourceAttribute.cs#L90

because this call via reflection is not aware of/smart enough to use the implicit conversion.